runtime/observability/telemetry-api/common/src/aws/smithy/kotlin/runtime/telemetry/context/TelemetryContextElement.kt

```diff
@@ -17,6 +17,11 @@ public expect class TelemetryContextElement(context: Context) : CoroutineContext
     public companion object Key : CoroutineContext.Key<TelemetryContextElement>
 
     public val context: Context
+    override val key: CoroutineContext.Key<*>
+    override fun <E : CoroutineContext.Element> get(key: CoroutineContext.Key<E>): E?
+    override fun <R> fold(initial: R, operation: (R, CoroutineContext.Element) -> R): R
+    override fun plus(context: CoroutineContext): CoroutineContext
+    override fun minusKey(key: CoroutineContext.Key<*>): CoroutineContext
 }
```

Comment:
**Question**: The [K2 migration guide shows](https://kotlinlang.org/docs/k2-compiler-migration-guide.html#stricter-rules-for-abstract-members-in-expected-classes) overriding abstract members with the `expect` keyword, which is not included here. Do you know what the difference is? Is `expect` an optional keyword for abstract members of `expect` classes or does it change the meaning/implementation somehow?